_pkgdown.yml

```diff
@@ -21,8 +21,8 @@ navbar:
       href: https://github.com/insightsengineering/teal.slice
 
 reference:
-  - title: Teal filter-panel API
-    desc: Functions used initialize filter-panel and to modify its states.
+  - title: "teal filter-panel API"
```

Comment:
Why remove the backticks?

Reply:
I have added backticks. Earlier I was getting some syntax issues on cran check but its not there anymore.